cllc-public-app/Controllers/LoginController.cs

```diff
@@ -15,49 +16,59 @@ namespace Gov.Lclb.Cllb.Public.Controllers
     public class LoginController : ControllerBase
     {
         private readonly IConfiguration _configuration;
+        private readonly IDynamicsClient _dynamicsClient;
         private readonly IWebHostEnvironment _env;
+        private readonly IHttpContextAccessor _httpContextAccessor;
         private readonly SiteMinderAuthOptions _options = new SiteMinderAuthOptions();
 
-        public LoginController(IConfiguration configuration, IWebHostEnvironment env)
+        public LoginController(IConfiguration configuration, IWebHostEnvironment env, IDynamicsClient dynamicsClient,
+            IHttpContextAccessor httpContextAccessor)
         {
             _configuration = configuration;
+            _dynamicsClient = dynamicsClient;
             _env = env;
+            _httpContextAccessor = httpContextAccessor;
         }
 
         [HttpGet]
         [Authorize]
         public ActionResult Login(string path, [FromQuery] string source)
         {
             // check to see if we have a local path.  (do not allow a redirect to another website)
-            if (!string.IsNullOrEmpty(path) && (Url.IsLocalUrl(path) || (!_env.IsProduction() && path.Equals("headers"))))
+            if (!string.IsNullOrEmpty(path) && (Url.IsLocalUrl(path) || !_env.IsProduction() && path.Equals("headers")))
             {
                 // diagnostic feature for development - echo headers back.
-                if ((!_env.IsProduction()) && path.Equals("headers"))
+                if (!_env.IsProduction() && path.Equals("headers"))
                 {
-                    ContentResult contentResult = new ContentResult();
+                    var contentResult = new ContentResult();
                     contentResult.Content = LoggingEvents.GetHeaders(Request);
                     contentResult.ContentType = "text/html";
                     return contentResult;
                 }
+
                 return LocalRedirect(path);
             }
 
-            string basePath = string.IsNullOrEmpty(_configuration["BASE_PATH"]) ? "/" : _configuration["BASE_PATH"];
+
+            // get the current user.
+            UserSettings userSettings = UserSettings.CreateFromHttpContext(_httpContextAccessor);
+
+            bool isSep = source != null && source == "sep" ||
+                userSettings?.ContactId != null &&
+                ContactController.IsSepPoliceRepresentative(userSettings?.ContactId, _configuration, _dynamicsClient);
+
+            var basePath = string.IsNullOrEmpty(_configuration["BASE_PATH"]) ? "/" : _configuration["BASE_PATH"];
             // we want to redirect to the dashboard.
-            string url = "dashboard";
-            if (source == "sep")
```

Comment:
There is corresponding code in cllc-public-app\ClientApp\src\app\components\sep\login\login.component.html line 17
that depends on this line of code. There is a sep login button that is supposed to land on the sep dashboard after login. Please adjust the other file to reflect the current changes.